src/validators/common/mongodb.validators.js

```diff
@@ -0,0 +1,23 @@
+import { body, param } from "express-validator";
+
+/**
+ *
+ * @param {string} idName
+ * @returns ValidationChain[]
```

Comment:
remove the `@returns ValidationChain[]` tag. Javascript is automatically inferring the return type.